### PR TITLE
Yar/ruby upgrate

### PR DESCRIPTION
**Pull request by yarhtut**
Acceptance Criteria
=================
 SJ Manager updated to Ruby 2.3

Background
=================
 Updated gems for ruby 2.3.0. Made changes for new version of Devise.

Checklist
=================

- [x] Code is understandable without Dev
- [x] Acceptance criteria, what was changed, and why it was changed (If applicable) on Pull / Merge Request
- [x] All Tests are Passing
- [x] Code Coverage goes up
- [x] All new methods have a relevant spec
- [x] Methods have a single responsibility (Where applicable)
- [ ] Linters Pass
- [x] ‘Yard’ style comments on methods and classes (Where applicable)

---
This change is [Reviewable](https://reviewable.io/reviews/digitalnz/supplejack_manager/9)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -59,17 +58,18 @@ group :development do
...
+  gem 'test-unit'
 end
 
 group :test do
   gem 'simplecov', require: false
-  gem 'database_cleaner', '>= 0.9.1'
+  gem 'database_cleaner', '>= 1.3.0'
   # gem 'email_spec', '>= 1.4.0'
-  gem 'cucumber-rails', '>= 1.3.0', :require => false
+  gem 'cucumber-rails', '>= 1.4.0', :require => false
```
Comment (Reinkaos) [inline Gemfile]: Use ruby 1.9 syntax, so

require: false

File: config/initializers/deprecations.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module Rack
```
Comment (Reinkaos) [inline config/initializers/deprecations.rb]: Is there a way that we can remove this?

**Comment by Reinkaos** [comment]
Looks good Yar.

**Comment by EdwinRozario** [comment]
Please fix the Rubocop offences for the .rb files changed.

**Pull request merged**